samples/apps/copilot-chat-app/webapp/env.example

```diff
@@ -1,2 +1,3 @@
 REACT_APP_BACKEND_URI=https://localhost:40443/
+REACT_APP_AAD_AUTHORITY=https://login.microsoftonline.com/common
```

Comment:
Since this is currently bound to AAD, convention would be to have this just be the tenant in place of the the whole authority URI.
```
REACT_APP_AAD_TENANT=common
```

Reply:
I was following the msal.js convention: https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/samples/msal-react-samples/react-router-sample/.env.development

They typically refer to authority and include the entire string:

<img width="773" alt="image" src="https://user-images.githubusercontent.com/146438/234369232-d456ebda-d412-450e-987e-98625f08c7a4.png">

Reply:
Cool - sounds good. I remember now that MSAL.js supports non-AAD while MSAL.NET supports only AAD, so that's probably where the friction is. Your changes looks good.